cli/src/cli.rs

```diff
@@ -10,14 +10,10 @@ use cubelib::steps::step::{StepConfig};
 #[command(author = "Jonas Balsfulland <cubelib@joba.me>")]
 #[command(version = "1.2")]
 pub struct Cli {
-    #[arg(short, long, default_value_t = false, group = "log_level", help = "Enables more detailed logging")]
-    pub verbose: bool,
-    #[arg(long, default_value_t = false, group = "log_level", help = "Prints nothing but the solutions")]
-    pub quiet: bool,
-    #[arg(id = "compact", short = 'c', long = "compact", default_value_t = false, help = "Prints only the solution, and not the different steps")]
-    pub compact_solutions: bool,
-    #[arg(short = 'p', long = "plain", default_value_t = false, requires = "compact", help = "Does not print the number of moves of the solution")]
-    pub plain_solution: bool,
+    #[arg(short, long = "log", default_value = "warn", help = "Log level: error|warn|info|debug|trace")]
```

Comment:
Clap has support for parsing enums directly. That way we don't have to deal with validating a string field, and clap also generates useful help messages when something goes wrong etc. (Same for the format flag below)

Reply:
Replaced with enums